### PR TITLE
italian translation

### PR DESCRIPTION
**Pull request by Alerika**
added italian translation, Hi can you  accept my PR for hacktoberfest? thank you dear <3

### CODE REVIEW

File: GameData/NotSoSimpleConstruction/Localization/ it-it.cfg [hunk 1]
```diff
@@ -0,0 +1,19 @@
+// it-it v1.0.0.0
+// Not So SimpleConstruction! (NSSC)
+// created: 23 Ott 2022
+// updated: 23 Ott 2022
+
+// License: GPLv3
+
```
Comment (zer0Kerbal) [inline GameData/NotSoSimpleConstruction/Localization/ it-it.cfg:7]: // Italian translation provided by [Alerika](https://github.com/Alerika)

**Review by zer0Kerbal — APPROVED** [review]
🎬 

_grazie_

**Comment by zer0Kerbal** [comment]
@taniwha - some more translated strings for you

**Comment by zer0Kerbal** [comment]
@Alerika _grazie_, you may delete your fork now if you wish. plenty more of my mods have open Italian translation issues. Happy Hacktoberfest/

🎃🦇👻

**Comment by Alerika** [comment]
My result is not accept .. why? little sad face

**Comment by zer0Kerbal** [comment]
> My result is not accept .. why? little sad face
@Alerika 

It was and was merged into `1.0.99.7-prerelease` branch - will be released with other updates within a couple hours.

Also would you also translate this (was missing from en-us.cfg)

		// #EL_Agency_title = Kairyuu Shipping
		// #EL_Agency_desc = Kairyuu Shipping started out as a pizza delivery kerbal with delusions of grandeur. However, behind everyone's backs he managed to acquire a fleet of bicycles and kerbals to ride them. One thing led to another and eventually that pizza delivery kerbal got involved in mining and constructions. He now wishes to corner the space freight market by being the the foundation of every space station and base.

**Comment by zer0Kerbal** [comment]
In fact the GitHub pages have already been updated to reflect your contribution

https://zer0kerbal.github.io/NotSoSimpleConstruction/Attributions.html
https://zer0kerbal.github.io/NotSoSimpleConstruction/Localizations.html

**Comment by Alerika** [comment]
oh i understand, thaks for everythink <3

**Comment by zer0Kerbal** [comment]
> oh i understand, thaks for everythink <3

@Alerika welcome, and I did also add you to the 'assignees'

**Pull request merged**